alerts_backend/python/app.py

```diff
@@ -45,45 +46,105 @@
+            Column("alert_id", Integer),
+            Column("fa_flight_id", Text),
+            Column("ident", Text),
+            Column("registration", Text),
+            Column("aircraft_type", Text),
+            Column("origin", Text),
+            Column("destination", Text)
+        )
 
 
-def create_table():
+def create_tables():
     """
-    Check if the tables exist, and if they don't create them.
+    Check if the table(s) exist, and if they don't create them.
     Returns None, raises exception if error
     """
     try:
-        # Create the table if it doesn't exist
+        # Create the table(s) if they don't exist
         metadata_obj.create_all(engine)
-        app.logger.info("Table successfully created (if not already created)")
+        app.logger.info("Table(s) successfully created (if not already created)")
     except exc.SQLAlchemyError as e:
-        # Since creation of table is a critical error, raise exception
-        app.logger.error(f"SQL error occurred during creation of table (CRITICAL - THROWING ERROR): {e}")
+        # Since creation of table(s) is a critical error, raise exception
+        app.logger.error(f"SQL error occurred during creation of table(s) (CRITICAL - THROWING ERROR): {e}")
         raise e
 
 
-def insert_into_db(data_to_insert: Dict[str, Union[str, int, bool]]) -> int:
+def insert_into_table(data_to_insert: Dict[str, Union[str, int, bool]], table: Table) -> int:
     """
-    Insert object into the database based off of the engine.
-    Assumes data_to_insert has values for all the keys:
-    fa_alert_id, ident, origin, destination, aircraft_type, start_date, end_date.
+    Insert object into the database based off of the table.
+    Assumes data_to_insert has values for all the keys
+    that are in the data_to_insert variable, and also that
+    table is a valid SQLAlchemy Table variable inside the database.
     Returns 0 on success, -1 otherwise
     """
     try:
         with engine.connect() as conn:
-            stmt = insert(aeroapi_alert_configurations)
+            stmt = insert(table)
             conn.execute(stmt, data_to_insert)
             conn.commit()
-
-            app.logger.info("Data successfully inserted into table")
-
+            app.logger.info(f"Data successfully inserted into table {table.name}")
     except exc.SQLAlchemyError as e:
-        app.logger.error(f"SQL error occurred during insertion into table: {e}")
+        app.logger.error(f"SQL error occurred during insertion into table {table.name}: {e}")
         return -1
-
     return 0
 
 
+@app.route("/post", methods=["POST"])
+def handle_alert() -> (Response, int):
+    """
+    Function to receive AeroAPI POST requests. Filters the request
+    and puts the necessary data into the SQL database.
+    Returns a JSON Response and also the status code in a tuple.
+    """
+    # Form response
+    r_title: str
+    r_detail: str
+    r_status: int
+    data: Dict[Any] = request.json
```

Comment:
Does mypy actually like this? I would think it'd be complaining about `Dict` not having 2 args (one for keys, one for values). Because it's coming from JSON which requires string keys I think the proper annotation would be `Dict[str, Any]`

Reply:
Never mind - made the typing changes in this PR